proposals/2021/Deprecate WallID.md

```diff
@@ -0,0 +1,32 @@
+# Deprecate WallID as direct child under Side
+
+## Overview
+
+This proposal is to deprecate the choice of single `auc:WallID` reference as direct child under `auc:Side`.
+
+## Justification
+
+`WallIDs` was added as an `xsd:choice` element in [#223](https://github.com/BuildingSync/schema/pull/223) to allow multiple `auc:WallID`s referenced for a `auc:Side` element (unbounded). This made the single `auc:WallID` reference choice redundant, so we propose to remove it.
+On the other hand, to avoid breaking change for Audit Template, the `xsd:choice` element will remain.
+
+## Implementation
+
+```xml
+                        <xs:element name="Side" maxOccurs="unbounded">
+                          <xs:complexType>
+                            <xs:sequence>
+                              ...
+                              <xs:choice>
```

Comment:
Thanks, looks like we need to remove choice here too

Reply:
You're right. I just update the proposal.